### ISSUE TITLE
Single card 4090 Why cannot the batch size be set to 24?

### ISSUE THREAD

**Issue by maominluck** [IDT-ITI/MMFusion-IML#15]
Hello,Thanks for your work very much.I have a question to ask, which is about Exploring Multi-Modal Fusion for Image
Manipulation Detection and Localization This paper. When training the positioning stage in the code of this paper, I set the maximum batch size size can only be 8 (in the case of using 4090 graphics card 24G), I read the paper that the batch size is 24. Here I have encountered a problem, I want to ask why this is?Looking forward to your reply.

**Comment by kostino** [comment]
Hi, in order to achieve the effective batch size of 24 we did gradient accumulation every 3 steps with a physical batch size of 3. The experiment files i have uploaded are the exact ones i used for training.

**Comment by kostino** [comment]
For late fusion we used batch size: 4, accumulate iters: 6
For early fusion we used batch size: 8, accumulate iters: 3

**Comment by maominluck** [comment]
Hello, thank you for your reply.Besides, I have another question I'd like
to ask you.Can you share the test data sets DSO-1 and CocoGlide? Because I
failed to download it online.Looking forward to your reply.


Wish you a happy life.
MM

Kostas Triaridis ***@***.***> 于2024年6月13日周四 07:14写道：
